File: src/main/java/Lesson_2/ChoseOperation.java
package Lesson_2;

public class ChoseOperation public class ChoseOperation extends Calculator{
    public ChoseOperation(double num1, double num2, String sym) {
        super(num1, num2, sym);
    }

    public String operation(String sym) {
        switch (sym) {
            case "+":
                return (String.format("%.2f + %.2f = %.2f", getNum1(), getNum2(), summa()));
            case "-":
                return String.format("%.2f - %.2f = %.2f", getNum1(), getNum2(), difference());
            case "*":
                return String.format("%.2f * %.2f = %.2f", getNum1(), getNum2(), multiply());
            case "/":
                return String.format("%.2f / %.2f = %.2f", getNum1(), getNum2(), devide());
            default:
                return null;
        }
    }

}

File: src/main/java/Lesson_7/ChoseOperation.py
from Calculator import Calculator


class ChoseOperation(Calculator):
    def __init__(self, first_number, second_number, symbol):
        self._first_number = first_number
        self._second_number = second_number
        self._symbol = symbol

    def operation(self,symbol):
        match symbol:
            case "+":
                return self.summa()
            case "-":
                return  self.difference()
            case "*":
                return self.multiply()

            case "/":
                return self.devide()
            case _:
                print("Invalid operator")